ccmbioinfo/lfs
# CARDS-2198: PROMS: Remove the restriction on audit_sex for tokenless access

To test:
- proms mode
- create a Patient Information form with gender either `male` or `female`
- create a Visit Information form for the Cardio clinic
- login (without a token) as the patient, fill in all the forms, selecting alcoholic answers to ensure a high AUDITC score
- submit the survey
- check that the summary for AUDITC says directly `If your symptoms result in a score...` and not `If you are a man:...`
